lib/ansible/module_utils/ns1.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, Matthew Burtless <mburtless@ns1.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
```

Comment:
module utils should be bsd licensed

Reply:
Thanks for the catch, missed that requirement from the docs